modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -485,6 +509,13 @@ public void processOpts() {
                         sourceFolder,
                         "URLSessionImplementations.swift"));
                 break;
+            case LIBRARY_VAPOR:
+                additionalProperties.put("useVapor", true);
+                typeMapping.put("object", "JSON");
```

Comment:
@aymanbagabas I think the change of `AnyType` and `object` to `JSON`, could be applied to all libraries.
What do you think?